File: app/src/main/java/br/com/brqtest/model/EntidadeDominio.java
package br.com.brqtest.model;

import java.io.Serializable;

public abstract class EntidadeDominio implements Serializable {

    private Integer id;


    public Integer getId() {
        return id;
    }

    @Override
    public String toString() {
        return "id=" + id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
}

File: app/src/main/java/br/com/brqtest/modelkotlin/EntidadeDominio.kt
package br.com.brqtest.modelkotlin

abstract class EntidadeDominio {
    private var id: Int? = null


    fun getId(): Int? {
        return id
    }

    override fun toString(): String {
        return "id=" + id!!
    }

    fun setId(id: Int?) {
        this.id = id
    }
}